velox/connectors/hive/HiveDataSink.h

```diff
@@ -384,6 +384,7 @@ struct HiveWriterIdEq {
   }
 };
 
+/// NOTE: this object is not thread-safe.
```

Comment:
Should this note go into connector interface. It would be good to document that DataSink implementation doesn't need to be thread safe.

Reply:
Yeah, move to there.